Epsilon/Export/Exporters/WordModuleExporter.cs

```diff
@@ -56,12 +56,13 @@ public async Task<Stream> Export(ExportData data, string format)
 
         var altChunkId = "HomePage";
 
-        var personaHTML = new HtmlDocument();
-        personaHTML.LoadHtml(data.PersonaHtml);
+        var personaHtml = new HtmlDocument();
+        personaHtml.LoadHtml(data.PersonaHtml);
 
+        var updatedPersonaHtml = await ReplaceImageSrcWithBase64String(personaHtml);
```

Comment:
This is not very clean, you're creating an HTML document just to create another one from it which results in unnecessary object allocations. I would personally prefer that this method "ReplaceImageSrcWithBase64String", is renamed to something like "GetPersonaHtmlDocument", which receives the persona HTML string and automatically handles the preprocessing of the Base64 images.